docs/modules/user-guide/partials/assembly_adding-commands-to-a-devfile.adoc

```diff
@@ -0,0 +1,24 @@
+ifdef::context[:parent-context-of-assembly_adding-commands-to-a-devfile: {context}]
+
+
+ifndef::context[]
+[id="assembly_adding-commands-to-a-devfile"]
+endif::[]
+ifdef::context[]
+[id="assembly_adding-commands-to-a-devfile_{context}"]
+endif::[]
+= Adding commands to a devfile
+
+[role="_abstract"]
+A devfile allows you to specify commands to run in a workspace. Every command can contain a subset of actions. These subsets are related to specific components and are run in the component containers.
```

Comment:
```suggestion
A devfile enables you [Or, "You can use a devfile..."] to specify commands to run in a workspace. Every command can contain a subset of actions. The actions in each subset are related to a specific component and run in the containers for that component.
```

Reply:
Saying "You can use a devfile..." gives more agency to the user by focusing on their actions and goals.